doc/headlines.txt

```diff
@@ -305,6 +308,11 @@ quote_string                                            *headlines-quote_string*
 
     Specifies the string that is used to display quotes.
 
+------------------------------------------------------------------------------
+list_marker_string                                         *list_marker_string*
+
+    Specifies the string that is used to display list markers.
```

Comment:
mention that it replaces all kinds of list items?

Reply:
What do you mean by `all kinds of list items`? All types of list item markers? 

IMHO, `list markers` already mean all.